pages/_includes/favicon.html

```diff
@@ -1,8 +1,7 @@
-<link rel="shortcut icon" type="image/ico" href="{{ '/assets/img/favicon/favicon.ico' | relative_url}}">
-<link rel="apple-touch-icon" sizes="180x180" href="{{ '/assets/img/favicon/apple-touch-icon.png' | relative_url}}">
-<link rel="icon" type="image/png" sizes="32x32" href="{{ '/assets/img/favicon/favicon-32x32.png' | relative_url}}">
-<link rel="icon" type="image/png" sizes="16x16" href="{{ '/assets/img/favicon/favicon-16x16.png' | relative_url}}">
-<link rel="manifest" href="{{ '/assets/img/favicon/site.webmanifest' | relative_url}}">
-<link rel="mask-icon" href="{{ '/assets/img/favicon/safari-pinned-tab.svg' | relative_url}}" color="#5bbad5">
+<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+<link rel="manifest" href="/site.webmanifest">
+<link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
```

Comment:
The favicon should show show up in the preview site. The URLs here are wrong, you probably want to revert them back to what they were before  e.g. `href="{{ '/assets/img/favicon/favicon-16x16.png' | relative_url}}"`

Reply:
Thanks so much @adborden! That did the trick. [Preview here](https://cg-f5a56614-c88e-470c-9f06-ff5fe2d4c98a.app.cloud.gov/preview/gsa/resources.data.gov/favicon-test/)